<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>464</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>18</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>--开启/重载--</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>14</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>roomId:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="roomLine"/>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>连接/重载</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disConnect">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>断开</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>18</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>--消息样式--</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>字体:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>字体大小:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fontSizeEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>弹幕显示窗体透明度：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="opacityEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>18</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>--设置屏蔽词--</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>屏蔽词:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="banWord"/>
         </item>
         <item>
          <widget class="QPushButton" name="addDeleteButton">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>加入/删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearWord">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="showBanWord">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>18</pointsize>
           <italic>false</italic>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>得意黑</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>--导出/导入配置--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="loadSetting">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>导入配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="outSetting">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>导出配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fixSetting">
           <property name="font">
            <font>
             <family>得意黑</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>修复/恢复默认配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
